src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -47,7 +33,8 @@ interface WorkPackageFormViewProps {
   leadOrManagerOptions: User[];
   blockedByOptions: { id: string; label: string }[];
   crId?: string;
-  createForm?: boolean;
+  formType: WPFormType;
+  schema: any;
```

Comment:
let's constrain this type as much as we can, should at least be able to make it this: `ObjectSchema<any>`